### DISCUSSION TITLE
Server Restart

### DISCUSSION THREAD

**Discussion by rogermungo** [inventree/InvenTree#6856] [Q&A] [answered]
Installed IvenTree

When I log in I see
Server Restart Required
A configuration option has been changed which requires a server restart. Contact your system administrator for further information 

What is the best or proper way to restart the server?

I've tried this
systemctl restart iventree
Failed to restart iventree.service: Unit iventree.service not found.

Which is strange because 
  inventree.service                     loaded    active   running inventree.service

**Comment by wolflu05** [comment, marked as answer]
You would need to provide more details about how you setup InvenTree. From your comments I can only guess you're using the package installer.

Please show us exactly what `systemctl status inventree` outputs. (You can restart it with `systemctl restart inventree`, again if you see an error, please provide the full error)

And note that it's not iventree, it's i**n**ventree. Maybe that's just a spelling issue.

**Reply by rogermungo** [reply]
Thanks. Spelling error.. How did I not spot that..
